src/hooks/UseToggle.jsx

```diff
@@ -0,0 +1,26 @@
+import { useEffect, useState } from "react";
+function UseToggle() {
+  const [Infor, setInfor] = useState([]);
+  const [isLoading, setIsLoading] = useState(false);
+
+  const fetchData = async () => {
+    try {
+      setIsLoading(true);
+      const response = await fetch(
+        "https://jsonplaceholder.typicode.com/users"
+      );
+      if (response.ok) {
+        const data = await response.json();
+        setInfor(data);
+        setIsLoading(false);
+      } else console.log("error");
```

Comment:
Inside the `else` clause, you should set `isLoading` to `false`, not doing so would make `isLoading` always `true` in case an error happens.
```js
if (response.ok) {
        const data = await response.json();
        setInfor(data);
        setIsLoading(false);
      } else {
        console.log("error");
        setIsLoading(false);
      }
```
or 
```js
if (response.ok) {
        const data = await response.json();
        setInfor(data);
      } else {
        console.log("error");
     }
    setIsLoading(false);
```